apps/mobile/src/store/apps/utils.ts

```diff
@@ -0,0 +1,24 @@
+import z from 'zod';
+
+const connectedAppSchema = z.object({
+  type: z.literal('connected'),
+  accountId: z.string(),
+});
+const recentlyVisitedAppSchema = z.object({
+  type: z.literal('recently_visited'),
+});
+
+const baseAppSchema = z.object({
+  origin: z.string(),
+  name: z.string(),
+  icon: z.string(),
```

Comment:
Where/how is this used?

Reply:
> Though I am thinking about changing the name to the homepage (leather.io) title.
So for leather.io it should be "Leather Bitcoin Wallet - Bitcoin for the Rest of Us"

Don't think we should do this, we want to pull as little as possible from the app as possible for security reasons.

Reply:
But on the icon, we derive the icon from the origin via the google api no? So why the record here?